src/pyird/utils/irdstream.py

```diff
@@ -545,15 +560,20 @@ def normalize1D(self,flatid='flat',master_path=None):
                 wfile = self.anadir/('wmmfmmf_%s_%s.dat'%(self.band,self.trace.mmf))
                 nwsave_path = self.anadir/('nwmmfmmf_%s_%s.dat'%(self.band,self.trace.mmf))
                 ncwsave_path = self.anadir/('ncwmmfmmf_%s_%s.dat'%(self.band,self.trace.mmf))
+                LFCpath = self.anadir/'wmmfmmf_y_m1.dat'
             else:
                 wfile = self.anadir/('w%d_%s.dat'%(id,self.trace.mmf))
                 nwsave_path = self.anadir/('nw%d_%s.dat'%(id,self.trace.mmf))
                 ncwsave_path = self.anadir/('ncw%d_%s.dat'%(id,self.trace.mmf))
-            df_continuum, df_interp = comb_norm(wfile,flatfile)
-            df_continuum_save = df_continuum[['wav','order','nflux']]
-            df_interp_save = df_interp[['wav','nflux']]
-            df_continuum_save.to_csv(nwsave_path,**self.tocsvargs)
-            df_interp_save.to_csv(ncwsave_path,**self.tocsvargs)
+                if self.band == 'h':
+                    LFC_path = self.anadir.parent/'y'/('w%d_%s.dat'%(id-1,'m1'))
+                elif self.band == 'y':
+                    LFC_path = self.anadir.parent/'y'/('w%d_%s.dat'%(id,'m1'))
```

Comment:
I think that it's better to set like `self.anadir/('w%d_%s.dat'%(id, 'm1'))` (I mean, not `self.anadir.parent/'y'`). (Or, add the option like `LFC_dir` allowing to set arbitrary by a user?)

Reply:
Oh, I agree with you. self.anadir/('w%d_%s.dat'%(id, 'm1')) will be OK. I put outputs in two folders ("y" and "h"). But it seems ok to put results from two bands together in the same folder.